diff --git a/apps/server/src/feeds/feeds.service.ts b/apps/server/src/feeds/feeds.service.ts
index 45727cd..3ec4caa 100644
--- a/apps/server/src/feeds/feeds.service.ts
+++ b/apps/server/src/feeds/feeds.service.ts
@@ -3 +2,0 @@ import { PrismaService } from '@server/prisma/prisma.service';
-import { Cron } from '@nestjs/schedule';
@@ -69,34 +67,0 @@ export class FeedsService {
-  @Cron(process.env.CRON_EXPRESSION || '35 5,17 * * *', {
-    name: 'updateFeeds',
-    timeZone: 'Asia/Shanghai',
-  })
-  async handleUpdateFeedsCron() {
-    this.logger.debug('Called handleUpdateFeedsCron');
-
-    const feeds = await this.prismaService.feed.findMany({
-      where: { status: 1 },
-    });
-    this.logger.debug('feeds length:' + feeds.length);
-
-    const updateDelayTime =
-      this.configService.get<ConfigurationType['feed']>(
-        'feed',
-      )!.updateDelayTime;
-
-    for (const feed of feeds) {
-      this.logger.debug('feed', feed.id);
-      try {
-        await this.trpcService.refreshMpArticlesAndUpdateFeed(feed.id);
-
-        await new Promise((resolve) =>
-          setTimeout(resolve, updateDelayTime * 1e3),
-        );
-      } catch (err) {
-        this.logger.error('handleUpdateFeedsCron error', err);
-      } finally {
-        // wait 30s for next feed
-        await new Promise((resolve) => setTimeout(resolve, 30 * 1e3));
-      }
-    }
-  }
-
